ceph_installer/controllers/osd.py

```diff
@@ -79,6 +81,7 @@ def configure_post(self):
             skip_tags="package-install",
             playbook="osd-configure.yml",
             verbose=verbose_ansible,
+            ceph_conf_overrides=request.json.get('conf', {}),
```

Comment:
Why are you passing this to the task? Doesn't this need to be in ``extra_vars`` only?